deploy/templates/nstemplatetiers/test/cluster.yaml

```diff
@@ -13,6 +13,7 @@ objects:
         count/deployments.apps: "30"
         count/deploymentconfigs.apps: "30"
         count/pods: "50"
+        count/virtualmachines.kubevirt.io: "2"
```

Comment:
Just FYI. We don't really have to keep the test tier up to date. It's not used anywhere atm. We have this tier just in case we need to test something. Then we can add something new to the test tier, merge it and then promote a test user to the test tier in prod.
But it doesn't hurt to update this tier of course.